wandb/sdk/artifacts/artifact.py

```diff
@@ -1590,23 +1621,36 @@ def _download_file(
             _thread_local_api_settings.cookies = cookies
             _thread_local_api_settings.headers = headers
 
-            self.get_path(name).download(root)
+            entry.download(root)
```

Comment:
I think you could get a big speedup here by re-using a session here.  I'm pretty sure `entry.download` just ends up calling `requests.get` which will create a new session which means it needs to re-negotiate the SSL certificate for every single file downloaded which is a bunch of overhead.  @shawnlewis worked on improving some of this in `weave` he may have some feedback / ideas.

Reply:
We are reusing a session, look for `self._session.get` in this PR.